services/portal/GWAS/GWASQuestions.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable max-len */
+const I = actor();
+
+const GWASTasks = require('./GWASTasks.js');
+const GWASProps = require('./GWASProps.js');
+
+module.exports = {
+  async isJobStart(jobId) {
```

Comment:
i think the following code can be refactored to take in the elem path they're waiting for

`jobHelper(elemPath) {
    let refreshed = 0; // keep track of how many refreshes
    while (refreshed < 3) {
      if (refreshed > 0) {
        I.refreshPage();
        await GWASTasks.CheckJobStatus();
      }
      let elemSpan = await tryTo(() => I.waitForElement(elempath, 2)); // eslint-disable-line
      if (!elemSpan) {
        refreshed += 1;
      } else {
        break;
      }
    }
    I.seeElement(elemPath);

}

isJobStart(jobId) {
const InProgressxpath = `//li[.//h4[contains(text(),"${jobId}")]]//span[text()="In Progress"]`;
jobHelper(InProgressxpath)
}

isJobComplete(jobId) {
  const completexpath = `//li[.//h4[contains(text(),"${jobId}")]]//span[text()="Completed"]`;
  jobHelper(completexpath)
}`